src/main/java/data/DataSet.java

```diff
@@ -41,4 +42,15 @@ public static ResponseInfo messageNotFoundResponse() {
             .setMessage("Pet not found");
     }
 
+    public static Order addingOrder() {
+
```

Comment:
In general, it is a good way to avoid empty lines after declaration of classes and methods, you can read more about code style [here](https://www.oracle.com/technetwork/java/codeconventions-150003.pdf)